packages/connect-agreement/src/thegraph/connector.ts

```diff
@@ -28,16 +28,25 @@ export function subgraphUrlFromChainId(chainId: number) {
   return null
 }
 
+type AgreementConnectorTheGraphConfig = {
```

Comment:
Do you mean for our connectors only, or as a type that would be exported by `connect-core` for any connector to extend the base config? The way I see it, a connector configuration is the responsibility of the connector defining it, even if they happen to be identical in our case because we follow the same conventions. What do you think?